services/vpc_manager/src/main/resources/application.properties

```diff
@@ -1,12 +1,10 @@
 server.port=9001
-
 #####Microservice url configuration######
 microservices.route.service.url=http://localhost:9003/vpcs/
 
 #####Ignite configuration######
 ignite.host=localhost
 ignite.port=10800
-ignite.thin.client.enable=true
```

Comment:
We should keep this configuration as thick client is not fully supported in K8s scenario.

Reply:
ok